File: airport_on_time_performance/v1/get/python/airport_on_time_performance.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    Will there be a delay in the JFK airport on the 1st of September?
    '''
    response = amadeus.airport.predictions.on_time.get(airportCode='JFK', date='2020-09-01')
    print(response.data)
except ResponseError as error:
    raise error

File: airport_on_time_performance/v1/get/Node SDK/airport_on_time_performance.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Will there be a delay in the JFK airport on the 1st of September?
    const response = await amadeus.airport.predictions.onTime.get({
      airportCode: "JFK",
      date: "2022-09-01",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
